dll/win32/shell32/lang/el-GR.rc

```diff
@@ -797,6 +797,7 @@ BEGIN
     IDS_DISCONNECT "Disconnect"
     IDS_OPENFILELOCATION "Open f&ile location"
     IDS_SENDTO_MENU "Se&nd To"
+    IDS_COPYASPATHMENU "Copy as path"
```

Comment:
No keyboard accelerator here? (some other translations have it)

Reply:
The original had them but I removed them because it was unknown if it would clash with other language string accelerators. And now I have been asked to remove the English one as well.

Reply:
The reason for that request is that we are already in super-clusterfuck-land regarding those accelerators, even for the very easy languages like en-US, en-GB, ko-ko, jp-jp and 3xchinese.
The situation is even worse than that for other languages, see for reference
![image](https://github.com/reactos/reactos/assets/33393466/015653fb-a3bf-4bfa-882f-17a2987ec135)

I just don't want to make our life even harder by adding accelerators that do not even exist on 2k3sp2!
I do hope you understand my motivation better now!

Reply:
I can understand that. But at the same time, any 3rd-party application can add its own entries here with whatever accelerator it wants. IContextMenu is probably supposed to deal with this somehow.

Reply:
> any 3rd-party application can add its own entries here with whatever accelerator it wants

Yes, but if a 3rd-party-application decides to not be compatible to 2k3sp2 by not caring or being sloppy, then it is the 3rd-party-apps decision/fault. And not ours.
But we should not punish those apps, that do offer proper accelerators with excellent compat towards our target by doing random stuff. Then we would break the apps, that are actually written correctly and with great care.

Reply:
Resolved. If you want to merge the PR now, merge it.